### PR TITLE
Rename concept of "ranked" mods to "eligible for PP"

### PR DESCRIPTION
**Pull request by bdach**
Because that's what it is now, and pretending otherwise has been [confusing users](https://github.com/ppy/osu/discussions/27719), and at this point, us too. The flag has nothing to do with whether scores are on the leaderboard or not.

https://github.com/ppy/osu/assets/20418176/ec1279a7-dac1-4767-96e3-a6fe441eb90d

osu_2024-03-25_11-08-36

### CODE REVIEW

**Comment by peppy** [comment]
Maybe.

But also I think that issue thread has validity – I really doubt we want that mod combo to show up at the top play of the beatmap on leaderboards. Thinking we probably want a default/implicit filter on leaderboards if we're going to keep storing scores liket hat.

**Comment by Walavouchey** [comment]
if you're open to more concise wording then i could suggest "awards pp" and "doesn't award pp"

**Comment by bdach** [comment]
> Thinking we probably want a default/implicit filter on leaderboards if we're going to keep storing scores liket hat.

Well this requires elaboration because I don't know what that means. What sort of filters? Which mods would it encompass? Are we going to have another flag for that too? What is it going to be called? Because this arguably is still valid despite that because it frees up "ranked" to be used for leaderboards.

**Comment by bdach** [comment]
Well do note that with this I am also trying to address the fact that our internal vernacular also uses "ranked" - as in the `scores` table column name - but that does not mean the same thing as "ranked mods" and this keeps coming up over and over. And I presume that this is the direction of less resistance than renaming the table column.

**Comment by smoogipoo** [comment]
I'm not really against the naming (except that "PP" is a very specific term and I would've called it something like "Performance" or "PerformanceRanking" instead), but I'm a bit worried about the amount of moving pieces happening at once, so I'm going to keep this on the back burner.

**Comment by bdach** [comment]
> amount of moving pieces

What would those be?

Put it that way: if this is gonna hang unmerged I'll close it like in a week so that it's just not stuck there. I mostly _really_ don't wanna be witness to another conversation wherein we're confused that the `ranked` flag on the scores table has nothing to do with "ranked" mods, but I can probably get over it.

**Comment by Kori3** [comment]
Can i just say that I'm really against this change.
The phrase "eligible for pp" is overextending something that players are already able to understand.

If this is meant to apply to new players or something, I don't think it's that difficult to make a connection between a ranked map and a ranked mod combination.

Plus, the text box thata appears when you hover over the "ranked/unranked" text already serves the function of explaining what this means.




It doesn't overextend a single word and, even if new players don't get the term at first, getting this text can easily help them understand it.

I don't think it's good for translation accessibility either, it makes it more friendly, yes, but like, players wouldn't really have an issue adding one word to their dictionary.
Even if this is meant to be more comprehensible or something, the text "ranked/unranked" already replaces the "score multiplier" (which in my opinion makes more sense to have than ranked/unranked) and that was way more reasoning than having any of the current texts.
(ranked/unranked is still a very reliable thing to have, but there probably was a better way to implement it than removing the score multiplier text)
k that's all i just had a lot to say over three words

**Comment by ominoussage** [comment]
I suggest that if the beatmap being played is not a ranked/approved map, it should display "Unranked (*insert beatmap status*)" instead.

**Comment by bdach** [comment]
I'm not looking to start a conversation here sorry. Closing due to apparent lack of support.

**Comment by Rekunan** [comment]
> Ranked to players already basically means "gives pp". Because "loved" doesn't give pp.

While this is true, I've seen a surprising amount of players be confused on this point for the new lazer mods, so I think it would be actually very beneficial to go through with this pr.

My guess is that players translate rankability differently in the context of the new lazer mods, so it seems right to remove the requirement of that translation layer for a massively better understanding as a whole.

**Comment by ominoussage** [comment]
@Rekunan
#27756 was implemented as a solution.

**Comment by alexialunya** [comment]
Yeah, I can see the confusion, but this wording does seem way too verbose. Not sure what to do.

#27756 doesn't exactly address this issue, it simply resolves confusion about a PP value still showing up for "Unranked" plays.
